File: Server/schemas/workshop_grade/workshop_grade_out.py
from pydantic import BaseModel, Field
from utils.constants import MAX_LENGTH_NAME, MIN_GRADE, MAX_GRADE

class WorkshopGradeOut(BaseModel):
    id_grade: int
    id_workshop: int
    workshop: str = Field(..., max_length=MAX_LENGTH_NAME)
    id_student: int
    rut_student: str
    semester: str
    year: int
    grade: int = Field(..., ge=MIN_GRADE, le=MAX_GRADE) 

File: Client/src/features/workshops/types/workshop_grade_out.ts
// workshop_grade_out.ts
// Interface de calificación de taller (output) basada en OUT del backend

export interface WorkshopGradeOut {
  id_grade: number
  id_workshop: number
  workshop: string
  id_student: number
  rut_student: string
  semester: string
  year: number
  grade: number
} 